### ISSUE TITLE
Adding Media Code to Single Comments Template

### ISSUE THREAD

**Issue by mayvn10** [ARKHAM-Enterprises/meteor-comments-ui#72]
@matteodem tried to do a pull request but was denied so I am logging an issue for it.

Your commentsSingleComments.html located in 

> lib/components/commentsSingleComment/commentsSingleComment.html

 is missing the media code below in the **bootstrap** and **ionic** templates:

```
{{#if media.content}}
<div class="content-media">
 {{{mediaContent}}}
</div>
{{/if}}
```

**Comment by matteodem** [comment]
What do you mean by denied?

**Comment by mayvn10** [comment]
I cloned your repo, created a branch for this fix, committed it, and then when I tried to push it, it said I was denied for the push. I'll look into it later but this should be a quick fix since it's just adding the media code to the 2 templates.

**Comment by matteodem** [comment]
you need to create a fork and then push it in your fork, after that you can create a pull request over the UI.

**Comment by mayvn10** [comment]
Ah, I figured I missed a step. Thanks I'll do this later today.

**Comment by matteodem** [comment]
This is fixed in the latest version `1.1.0`, see https://github.com/ARKHAM-Enterprises/meteor-comments-ui/blob/master/GUIDE.md
